a10_octavia/db/models.py

```diff
@@ -43,9 +43,11 @@ class VThunder(base_models.BASE):
     compute_id = sa.Column(sa.String(36))
     topology = sa.Column(sa.String(50))
     role = sa.Column(sa.String(50))
+    last_update = sa.Column(sa.DateTime, default=func.now(), nullable=False)
```

Comment:
Change attribute name - "last_update" as its very similar to updated_at - can be changed to "last_udp_update"